Team2 food recipe

Joanna-Wang105 opened the pull request:
Create recipe.json under assets/configs
Create CookingConfig.java under entities/configs
Create DishFactory.java under entities/factories

madeleinecoombs commented:
Looks good! Merging now to start integrating the recipe.json with our meal item/s .

madeleinecoombs reviewed (commented):
Looks good!

Joanna-Wang105 commented:
@madeleinecoombs I've changed the name inside the recipe.json to **camal case** 
The one I changed were: acai bowl -> acaiBowl, cutting board -> cuttingBoard, making time -> makingTime, fruit salad -> fruitSalad, steak meal -> steakMeal, banana split -> bananaSplit
I've also changed the structure of the CookingConfig (don't think you will need this tho). Will do another pull request asap to update these changes. In the meantime, please feel free to look at team 2 branch to see the latest version of the json file :)
